src/main/java/androidx/view/View.kt

```diff
@@ -172,3 +173,19 @@ fun View.toBitmap(config: Bitmap.Config = Bitmap.Config.ARGB_8888): Bitmap {
     }
     return Bitmap.createBitmap(width, height, config).applyCanvas(::draw)
 }
+
+/**
+ * Sets the margins in the ViewGroup's MarginLayoutParams. This version of the method sets all axes
+ * to the provided size.
+ *
+ * @see ViewGroup.MarginLayoutParams.setMargins
+ * @throws IllegalStateException if view doesn't have layoutParams as  ViewGroup.MarginLayoutParams
+ */
+fun View.setMargins(@Px size: Int) {
```

Comment:
This feature doesn't belong on View. A view should never query its layout params.